Dockerfile.nginx

```diff
@@ -54,15 +54,14 @@ RUN pip3 install \
 	-r ./requirements.txt
 
 COPY components/ ./components/
+RUN \
+  cd components && \
+  yarn
+
 COPY manage.py ./
```

Comment:
was there a reason to move this part up? not sure, but it might actually help with better caching of docker layers

Reply:
Yes. it help a lot. because our npm dependencies doesn't change at all. So to have them before the copy of Dojo files let the cache use this layer. It take few minutes to build it.

Reply:
thanks. I had this on my list for Dockerfile optimizations.